esp8266-fastled-webserver/common.h

```diff
@@ -282,10 +279,11 @@ void radarSweepPalette();
 void radiusPalette();
 void angleGradientPalette();
 void radiusGradientPalette();
+void drawAnalogClock();
```

Comment:
Are these being used outside of map.cpp?  If not, it's probably better not to define them in `common.h`.

Reply:
Yes, as a pattern in the list: https://github.com/jasoncoon/esp8266-fastled-webserver/blob/a64f66aedfc824795f899ded4fc71e7a27d5333a/esp8266-fastled-webserver/esp8266-fastled-webserver.ino#L175

And here, if showClock is true: https://github.com/jasoncoon/esp8266-fastled-webserver/blob/a64f66aedfc824795f899ded4fc71e7a27d5333a/esp8266-fastled-webserver/esp8266-fastled-webserver.ino#L734